src/ui/issuepanel/IssuePanel.java

```diff
@@ -218,6 +230,15 @@ public void handle(KeyEvent event) {
 						filterTextField.requestFocus();
 					}
 				}
+				if (maximizeWindow.match(event)) {
+					ui.maximizeWindow();
+				}
+				if (minimizeWindow.match(event)) {
+					ui.minimizeWindow();
+				}
+				if (defaultSizeWindow.match(event)) {
+					ui.setDefaultWidth();
+				}
```

Comment:
Could we factor these (and the hotkeys above) out into another function? That way the hotkeys for the panel and the filter box would always stay the same.

Also, using `if (...) {...} else if (...) {...}` seems better here, as it makes it clear that only one of these conditions can occur at a time.